lightning-invoice/src/de.rs

```diff
@@ -715,15 +715,15 @@ impl From<ParseError> for ParseOrSemanticError {
 	}
 }
 
-impl From<::SemanticError> for ParseOrSemanticError {
+impl From<crate::SemanticError> for ParseOrSemanticError {
 	fn from(e: SemanticError) -> Self {
 		ParseOrSemanticError::SemanticError(e)
 	}
 }
 
 #[cfg(test)]
 mod test {
-	use de::ParseError;
+	use crate::de::ParseError;
```

Comment:
Could use `super` here and in quite a few places below.
```suggestion
	use super::ParseError;
```

Reply:
Yeah, fair enough.